pytest_splunk_addon/standard_lib/cim_tests/test_templates.py

```diff
@@ -247,57 +247,157 @@ def test_cim_fields_not_allowed_in_props(
     @pytest.mark.splunk_searchtime_cim
     @pytest.mark.splunk_searchtime_cim_mapped_datamodel
     def test_eventtype_mapped_multiple_cim_datamodel(
-        self, splunk_search_util, record_property, caplog
+        self,
+        splunk_search_util,
+        splunk_searchtime_cim_mapped_datamodel,
+        record_property,
+        caplog,
     ):
         """
         This test case check that event type is not be mapped with more than one data model 
 
         Args:
             splunk_search_util (SearchUtil): Object that helps to search on Splunk.
+            splunk_searchtime_cim_mapped_datamodel: Object which contain eventtype list
             record_property (fixture): Document facts of test cases.
             caplog (fixture): fixture to capture logs.
         """
 
         data_models = [
-            "Alerts",
-            "Authentication",
-            "Certificates",
-            "Change",
-            "Compute_Inventory",
-            "DLP",
-            "Databases",
-            "Email",
-            "Endpoint",
-            "Event_Signatures",
-            "Interprocess_Messaging",
-            "Intrusion_Detection",
-            "JVM",
-            "Malware",
-            "Network_Resolution",
-            "Network_Sessions",
-            "Network_Traffic",
-            "Performance",
-            "Splunk_Audit",
-            "Ticket_Management",
-            "Updates",
-            "Vulnerabilities",
-            "Web",
+            {"name": "Alerts", "tags": [["alert"]]},
+            {
+                "name": "Authentication",
+                "tags": [
+                    ["authentication"],
+                    ["authentication", "default"],
+                    ["authentication", "insecure"],
+                    ["authentication", "privileged"],
+                    ["authentication", "privileged"],
```

Comment:
Can we remove duplicate tags: "authentication", "privileged"

Reply:
Removed